### PR TITLE
Manage team on seller dashboard

### PR DESCRIPTION
**Pull request by Blackfry**
buyer dashboard team view
updated to slightly newer -v of react which fixed a testing error

### CODE REVIEW

File: src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+
+export const DashboardBriefs = ({live, closed, draft, all}) => (
+  <div className="content-main">
+    <DraftBriefs draft={draft}/>
+    <div>
+      <p>
+        <a href={(!isEmpty(all) ?
+          "/buyers/frameworks/".concat(
+            all[0].frameworkSlug,
```
Comment (maxious) [inline src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js]: This probably won't work for new teams without any "all" briefs or old teams who have only had DSP framework briefs but should now only make DM framework briefs?

File: src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+import './DashboardBriefs.css'
+
+
+export const DashboardBriefs = ({live, closed, draft, all}) => (
+  <div className="content-main">
+    <DraftBriefs draft={draft}/>
+    <div>
+      <p>
+        <a href={(!isEmpty(all) ?
+          "/buyers/frameworks/digital-marketplace/requirements/digital-professionals"
```
Comment (matt-sm) [inline src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js]: we could have passed these urls as props

File: src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+)
+
+DashboardBriefs.propTypes = {
+  live: React.PropTypes.array,
+  closed: React.PropTypes.array,
+  draft: React.PropTypes.array,
+  all: React.PropTypes.array
+}
+
+export const DraftBriefs = ({draft}) => (
```
Comment (matt-sm) [inline src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js]: put these components in separate files?

File: src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    )}
+  </section>
+)
+
+DraftBriefs.propTypes = {
+  draft: React.PropTypes.array.isRequired
+}
+
+
+export const ClosedBriefs = ({closed}) => (
```
Comment (matt-sm) [inline src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js]: is there a lot of copy/paste going on between these 3?  could they be 1 component?

File: src/bundles/BuyerDashboard/components/Views/Views.js [hunk 5]
```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import isEmpty from 'lodash/isEmpty';
+import format from 'date-fns/format';
+
+import './Views.css'
+
+
+export const DashboardBriefs = ({live, closed, draft, all}) => (
```
Comment (matt-sm) [inline src/bundles/BuyerDashboard/components/Views/Views.js]: is this a copy/paste of above?

File: src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js (hunk 4 above)
Comment (Blackfry) [inline src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.js]: some of the table headers and other text is different between them but they could be combined

File: src/bundles/BuyerDashboard/components/Views/Views.js (hunk 5 above)
Comment (Blackfry) [inline src/bundles/BuyerDashboard/components/Views/Views.js]: That whole folder was redundant after I split out the above and I deleted it but it one of my follow up commits it came back. Will remove.

**Pull request merged**